codeland.html

```diff
@@ -631,12 +646,14 @@
   });
 }
 
-//one time initial data load
-fetchScheduleData(SCHEDULE_DATA_URL);
+//one time initial data load, then kick off main polling loops
+fetchScheduleData(SCHEDULE_DATA_URL)
+  .then(updateNowPlaying);
```

Comment:
If the suggestion above is implemented there would be no need to chain `updateNowPlaying` here, since it's implicitly called everytime the schedule is fetched/refreshed :)

Reply:
Brilliant!